pom.xml

```diff
@@ -0,0 +1,150 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright 2024 Open Text.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.github.cafapi</groupId>
+        <artifactId>caf-common-parent</artifactId>
+        <version>5.0.0-431</version>
+    </parent>
+
+    <groupId>com.github.cafapi.kubernetes</groupId>
+    <artifactId>caf-kubernetes-client-aggregator</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+    <packaging>pom</packaging>
+
+    <name>CAF Kubernetes Client Aggregator</name>
+    <description>CAF Kubernetes Client Aggregator</description>
+    <url>https://github.com/CAFapi/caf-kubernetes-client</url>
+
+    <inceptionYear>2024</inceptionYear>
+
+    <licenses>
+        <license>
+            <name>Apache License, Version 2.0</name>
+            <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+        </license>
+    </licenses>
+
+    <developers>
+        <developer>
+            <id>dermot-hardy</id>
+            <name>Dermot Hardy</name>
+            <email>dhardy2@opentext.com</email>
+        </developer>
+        <developer>
+            <id>rorytorneymf</id>
+            <name>Rory Torney</name>
+            <email>rtorney@opentext.com</email>
+        </developer>
+    </developers>
+
+    <scm>
+        <connection>scm:git:https://github.com/CAFapi/caf-kubernetes-client.git</connection>
+        <developerConnection>scm:git:https://github.com/CAFapi/caf-kubernetes-client.git</developerConnection>
+        <url>https://github.com/CAFapi/caf-kubernetes-client</url>
+    </scm>
+
+    <modules>
+        <module>caf-kubernetes-client</module>
+    </modules>
+
+    <properties>
+        <copyrightYear>2024</copyrightYear>
+        <copyrightNotice>Copyright ${copyrightYear} Open Text.</copyrightNotice>
+        <kubernetesOpenApiSpecVersion>release-1.31</kubernetesOpenApiSpecVersion>
```

Comment:
I assume this means the latest `1.31` patch version?  Could we lock this down to `v1.31.0` instead, to avoid the possibility of it changing on us?